### DISCUSSION TITLE
Missing daily subtotals/summary in 2.x

### DISCUSSION THREAD

**Discussion by winternet-studio** [kimai/kimai#4417] [General] [answered]
I must say that I do miss the daily subtotal rows in the Time Tracking > My times listing... Version 1.x had them, but are gone in 2.x.

I use that every day to quickly glance over how much I have worked. This is the page that I use all the time and it will have to be annoying to go to another page several times a day to check how much time I have worked. Should at least be an option to have that summary row.

**Comment by kevinpapst** [comment, marked as answer]
Should be at least an option to RTM 😁 

https://www.kimai.org/documentation/user-preferences.html#show-daily-statistics-in-timesheet

**Comment by winternet-studio** [comment]
Oh there was an option for that already! I had already checked the preferences earlier but didn't find it then.
